### DISCUSSION TITLE
Separate Admin User and User in two different Tables

### DISCUSSION THREAD

**Discussion by lam0819** [filamentphp/filament#4050] [Ideas]
Currently we use same "users" table to login and do any auth.
I suggest separate admin user and normal web user in two different table.

- better manage
- easy to do additional fields on both admin user and web user
- either one user type change will not affect another one.

Here is another project I am using, they can separate two admin panels with two set tables too.
![image](https://user-images.githubusercontent.com/68211972/190651523-ebf66433-0f20-43f9-8c91-79e8f26c3446.png)

**Comment by ArtMin96** [comment]
Why? Not everyone separates users like that. Maybe this is what you want? [Filament Multi Context](https://filamentphp.com/plugins/multi-context)

**Reply by lam0819** [reply]
Maybe my common usage/concept, I don't combine user and admin user into the same table. 
I will look into the multi context first.

**Comment by ziming** [comment]
u can consider the parental package by caleb

**Reply by lam0819** [reply]
which package?

**Reply by leolll** [reply]
Incase anyone is interested, this is the Parental package (created Caleb Porzio & maintained by Tighten):  https://github.com/tighten/parental

**Comment by neuroticimbecile** [comment]
It's been a few years, but this might help someone:
https://medium.com/@josephajibodu/change-laravel-authentication-model-for-filamentphp-3-b69fa1560cb2

**Reply by lam0819** [reply]
we already done simiar things 😎， especially different panel with different user

**Reply by riyen-perdana** [reply]
can you show to me how to do it ??
